.github/workflows/js-build-and-lint.yml

```diff
@@ -19,6 +19,8 @@ jobs:
           # because we want to lint + fix + commit, we need to checkout the HEAD sha (otherwise
           # it checks out the merge commit and we will not be able to commit to it)
           ref: ${{ github.event.pull_request.head.ref || 'master' }}
+          # We need the repo here so that this works for forks
+          repo: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
Wait, don't we need `pull_request_target` à la https://github.com/getsentry/sentry/pull/21600 as well?

Reply:
We don't because there are no secrets that are used, and no write access to `getsentry/sentry` repo is needed.

Reply:
You can see it working here https://github.com/getsentry/sentry/pull/21906

Reply:
@joshuarli Updated to have a specific task for forks.